### ISSUE TITLE
Update default astrometry catalogs used for a posteriori processing

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#931]
_Issue [HLA-486](https://jira.stsci.edu/browse/HLA-486) was created on JIRA by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack):_

New astrometric catalogs get added to the guide star catalog web service.  Recently, the Early-Release of the GAIA DR3 catalog (labelled, GAIAedr3) was added as a new catalog to the web service.  A new Pan-STARRS catalog which has been fit to the GAIADR2 catalog as well as the full GAIA DR3 release will be added to the web service in the near future.  

These additions should be included as high priority options when performing *a posteriori* processing during standard pipeline calibration processing with **AstroDrizzle**.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Michael Dulude](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dulude) on [JIRA](https://jira.stsci.edu/browse/HLA-486?focusedCommentId=518140#comment-518140):_

```
All mentions of "GSC" anywhere in the code:
haputils/astrometric_utils.py:403:def find_gsc_offset(image, input_catalog='GSC1', output_catalog='GAIA'):
hapsequencer.py:941:    # Every image should (!) have an IDC_?????????-GSC240 solution.
hapsequencer.py:942:    wcs_preference = ['IDC_?????????-GSC240', 'IDC_?????????']
haputils/astrometric_utils.py:273:def get_catalog(ra, dec, sr=0.1, epoch=None, catalog='GSC241'):
haputils/astrometric_utils.py:298:        Name of catalog to query, as defined by web-service.  Default: 'GSC241'
haputils/astrometric_utils.py:328:def get_catalog_from_footprint(footprint, epoch=None, catalog='GSC241'):
haputils/astrometric_utils.py:346:        Name of catalog to query, as defined by web-service.  Default: 'GSC241'
haputils/astrometric_utils.py:403:def find_gsc_offset(image, input_catalog='GSC1', output_catalog='GAIA'):
runastrodriz.py:149:apriori_priority = ['HSC', 'GSC', '']
runastrodriz.py:1096:def verify_gaia_wcsnames(filenames, catalog_name='GSC240', catalog_date=gsc240_date):
runastrodriz.py:1096:def verify_gaia_wcsnames(filenames, catalog_name='GSC240', catalog_date=gsc240_date):```

**Comment by stsci-hack** [comment]
Resolved with #986.
